src/handlers/quickstart/quickstart-handler.ts

```diff
@@ -2,19 +2,79 @@ import { ConfigService } from '../../services/config/config.service';
 import { Logger } from '../../services/logger/logger.service';
 import { cleanExit } from '../clean-exit.handler';
 import { QuickstartSsmService } from '../../services/quickstart/quickstart-ssm.service';
-import { ValidSSHHost } from '../../services/quickstart/quickstart-ssm.service.types';
 import { MixpanelService } from '../../services/mixpanel/mixpanel.service';
-import { ParsedTargetString, TargetType } from '../../services/common.types';
 import { EnvironmentService } from '../../services/environment/environment.service';
 import { PolicyService } from '../../services/policy/policy.service';
-import { connectHandler } from '../connect/connect.handler';
 import { readFile } from '../../utils';
 import { SsmTargetSummary } from '../../services/ssm-target/ssm-target.types';
 import { defaultSshConfigFilePath, quickstartArgs } from './quickstart.command-builder';
+import { OAuthService } from '../../services/oauth/oauth.service';
+import { UserSummary } from '../../services/user/user.types';
+import { version } from '../../../package.json';
 
-import prompts from 'prompts';
+import prompts, { PromptObject } from 'prompts';
 import yargs from 'yargs';
 import fs from 'fs';
+import { ConsoleWithTranscriptService } from '../../services/consoleWithTranscript/consoleWithTranscript.service';
+import chalk from 'chalk';
+import { TranscriptMessage } from '../../services/consoleWithTranscript/consoleWithTranscript.types';
+import ora from 'ora';
+import { login } from '../login/login.handler';
+import { KeySplittingService } from '../../../webshell-common-ts/keysplitting.service/keysplitting.service';
+
+const welcomeMessage = `Welcome to BastionZero and the journey to zero trust access via our multi root zero trust access protocol (MrZAP). We're excited to have you!\n
+Our quickstart installer is a fast and easy method for you to try BastionZero using your existing SSH configuration.
+We will use the following information supplied by your SSH configuration file to onboard the targets:\n
+Host ${chalk.bold.yellowBright('your_host_name')}
+    HostName ${chalk.bold.yellowBright('your.ip.address')}
+    User ${chalk.bold.yellowBright('your_username')}
+    IdentityFile ${chalk.bold.yellowBright('path/to/your/.ssh/.pem/file')}
+    Port ${chalk.bold.yellowBright('specify_port_number_if_desired')}
+`;
+
+const loginMessage = `Below, when you press the key to continue a browser window will appear.\nAuthenticate yourself with Google or Microsoft based on your identity provider. Doing so will automatically create your BastionZero account.\nPress any key to continue...`;
```

Comment:
```suggestion
const loginMessage = `Below, when you press the key to continue a browser window will appear.\nLogin with your SSO provider. Doing so will automatically create your BastionZero account.\nPress any key to continue...`;
```

Not a huge deal but maybe we should future-proof the wording here to not be specific to google/microsoft

Reply:
ea4c2f79e5b8e0be225950fc01ed85ad70f93470